virtualbox/vbm.go

```diff
@@ -99,6 +100,11 @@ func getHostOnlyNetworkInterface(mc *driver.MachineConfig) (string, error) {
 
 	for _, n := range nets {
 		if dhcp, ok := dhcps[n.NetworkName]; ok {
+            fmt.Printf("IPv4 / DHCP IP: %s == %s\n", dhcp.IPv4.IP, mc.DHCPIP)
```

Comment:
yes, `boot2docker init` should set everything up, so if its not working for vbox5, it needs to be fixed too.